.gitignore

```diff
@@ -10,9 +10,10 @@ scorecard
 
 # Test binary, built with `go test -c`.
 *.test
+results.json
```

Comment:
slightly prefer to create a output directory and keeping test results in that dir rather than polluting root of tree.

Reply:
Took care of it.